src/auth/auth.ts

```diff
@@ -91,8 +95,18 @@ app.all("/logout", (request, response) => {
 	}
 });
 
-app.use(passport.initialize());
-app.use(passport.session());
+authRouter.get("/resend/:uuid", validateAndCacheHostName, async (request, response) => {
+	let user = await User.findOne({ uuid: request.params.uuid || "" });
+	if (user) {
+		await sendVerificationEmail(request, user);
+		const email = user.email
+			.replace(/</g, "&lt;")
+			.replace(/>/g, "&gt;")
+			.replace(/&/g, "&amp;");
+		request.flash("success", `Resent a verification email to ${email}. ${resendVerificationEmailLink(request, user.uuid)}`);

```

Comment:
This may reveal email addresses to anyone who knows a user's UUID

Reply:
Ok true but this will only appear if that account does not have a verified email yet (which shouldn't be applicable to many accounts). Also, if they haven't gotten an email, maybe they're checking the wrong email account so I think we should emphasize this. UUIDs aren't common knowledge anyway.

Reply:
Fair enough